gradle/libs.versions.toml

```diff
@@ -37,6 +38,7 @@ android-documentation-plugin = { module = "org.jetbrains.dokka:android-documenta
 androidx-activity-compose = { group = "androidx.activity", name = "activity-compose", version.ref = "androidx-activity" }
 # https://androidx.tech/artifacts/appcompat/appcompat-resources/
 androidx-appCompat-resources = { module = "androidx.appcompat:appcompat-resources", version.ref = "androidx-appCompat" }
+androidx-core = { module = "androidx.core:core-ktx", version.ref = "androidx-core" }
```

Comment:
It seems we don't need `core-ktx` since only `androidx.window:window` appears in the runtime classpath. Can we change this dependency?

Reply:
Yep I will change it 👍